constants.js

```diff
@@ -63,6 +63,11 @@ export default {
     // http://docs.aws.amazon.com/AmazonS3/latest/API/mpUploadComplete.html
     minimumAllowedPartSize: 5242880,
 
+    // Max size on put part or copy part is 5GB. For functional
+    // testing use 100 MB as max
+    maximumAllowedPartSize: process.env.MPU_TESTING === 'yes' ? 104857600 :
```

Comment:
I remember of a bad thing I did regarding testing code in production code. does it apply here?